config/rbac/role.yaml

```diff
@@ -21,6 +21,7 @@ rules:
   - ""
   resources:
   - namespaces
+  - secrets
```

Comment:
If the secret is always in the operator's namespace, you might not need this.

Reply:
Ah, is this is so that you can create the copy in the operand's namespace?

Reply:
Yes, it's for the operand namespace's secret.
Technically we need the access to 1 namespace only but this namespace is not fixed but configurable via a flag.